package.json

```diff
@@ -52,8 +52,11 @@
     "ava": "^0.17.0",
     "chokidar": "^1.6.1",
     "electron-connect": "^0.6.1",
-    "standard": "^8.6.0",
-    "uuid": "^3.0.1"
+    "expand-range": "^1.8.2",
+    "is-extendable": "^0.1.1",
+    "preserve": "^0.2.0",
+    "repeat-element": "^1.1.2",
+    "standard": "^8.6.0"
```

Comment:
hmm, it works for me. try the ol' `rm -rf node_modules` and `npm install` trick.